### PR TITLE
pull

### PR DESCRIPTION
**Pull request by EmilioFu**

### CODE REVIEW

**Review by liberhe — COMMENTED** [review]
you may move it docs/issue.md

File: nft/envbuid/readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+简易的开发环境搭建
```
Comment (maroonstar) [inline nft/envbuid/readme.md:1]: 大标题使用一个井号

`# 简易的开发环境搭建`

**Review by maroonstar — CHANGES_REQUESTED** [review]
内容看起来和NFT无关

**Comment by yingjingyang** [comment]
the doc format is wrong，need to be fixed

**Pull request closed without merging**